cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/asset_transfers.rs

```diff
@@ -1478,9 +1478,6 @@ fn send_pens_and_wnds_from_penpal_westend_via_ahw_to_ahr() {
 		);
 	});
 
-	// assert unique topic across all chains
-	TopicIdTracker::assert_unique();
```

Comment:
you should not remove this, e.g. you do `TopicIdTracker::insert("AssetHubRococo", mq_prc_id.into());` above so you need a final final check after all chains have processed their XCM messages, that all of them saw the same TopicId.

(same for the other tests)